### PR TITLE
MediaCollectionViewCell: use bigger thumbnails on ipad

### PR DESCRIPTION
**Pull request by carolanitz**
### Checklist
- [x] I've run `bundle exec fastlane test` from the root directory to see all new and existing tests pass
- [x] I've followed the [vlc-ios code style](Docs/CodingStyle.md)
- [x] I've read the [Contribution Guidelines](https://github.com/videolan/vlc-ios#contribute)
- [x] I've updated the documentation if necessary.

### CODE REVIEW

File: Sources/MediaCategoryCells/MediaCollectionViewCell.swift [hunk 1]
```diff
@@ -18,6 +18,7 @@ class MediaCollectionViewCell: BaseCollectionViewCell {
     @IBOutlet weak var titleLabel: UILabel!
     @IBOutlet weak var descriptionLabel: UILabel!
     @IBOutlet weak var newLabel: UILabel!
+    @IBOutlet weak var thumbnailWidth: NSLayoutConstraint!
```
Comment (Mikanbu) [inline Sources/MediaCategoryCells/MediaCollectionViewCell.swift]: Ideally, I believe the IBOutlet should be set as private.

File: Sources/MediaCategoryCells/MediaCollectionViewCell.swift [hunk 2]
```diff
@@ -43,6 +44,8 @@ class MediaCollectionViewCell: BaseCollectionViewCell {
         super.awakeFromNib()
         newLabel.text = NSLocalizedString("NEW", comment: "")
         newLabel.textColor = PresentationTheme.current.colors.orangeUI
+         let isIpad = UIDevice.current.userInterfaceIdiom == UIUserInterfaceIdiom.pad
```
Comment (Mikanbu) [inline Sources/MediaCategoryCells/MediaCollectionViewCell.swift]: Nit-pick: Extra space

**Comment by Mikanbu** [comment]
I was also wondering, shouldn't the images, colour(theme) be done before showing it to the users? 🤔 

For example, if we switch to the dark theme, we can see the default images change.

File: Sources/MediaCategoryCells/MediaCollectionViewCell.swift (hunk 1 above)
Comment (carolanitz) [reply Sources/MediaCategoryCells/MediaCollectionViewCell.swift]: can do and will do for all then

**Comment by carolanitz** [comment]
rebased and adjusted. the last one regarding you can see the default image change wasn't fixed. i can create a follow up ticket for that

**Comment by carolanitz** [comment]
merged with 35eebb8ad19a126269e26bc6101090c82f5b2cff

**Pull request closed without merging**